test/jdk/com/sun/nio/sctp/SctpMultiChannel/SocketOptionTests.java

```diff
@@ -38,6 +39,8 @@
 import java.nio.channels.ClosedChannelException;
 import com.sun.nio.sctp.AbstractNotificationHandler;
 import com.sun.nio.sctp.Association;
+import com.sun.nio.sctp.SctpStandardSocketOptions.InitMaxStreams;
```

Comment:
Was this intentional? Few lines below, there's already a:

```java
import static com.sun.nio.sctp.SctpStandardSocketOptions.*;
```

Reply:
Maybe the redundant import added by IDE automatically. I have deleted the unnecessary import and verified it locally. Thanks your reviews.